### ISSUE TITLE
Task list does not work with screen readers

### ISSUE THREAD

**Issue by benrhughes** [benrhughes/todotxt.net#108]
From Amy 

> Unfortunately the application is not revealing the text in the list of tasks to my screen access software. (NVDA or JAWS).
> ...
> All the other controls work well.
